### DISCUSSION TITLE
2FA or MFA support

### DISCUSSION THREAD

**Discussion by karvounis** [rotki/rotki#4404] [Ideas]
I recently realised that Rotki does not support 2-Factor Authentication (or Multi-Factor). Is that something that you think could be added to Rotki? I believe that it will increase the security of the users. Cheers

**Comment by kelsos** [comment]
Hey, @karvounis I am not sure how a 2FA/MFA would work on rotki technically.

One thing to keep in mind is that the rotki app account password is essentially the decryption key for the encrypted SQLite (SQLCipher) database that stores all your information.

I am not sure how a 2FA/MFA would work in such a case.

**Comment by LefterisJP** [comment]
2FA involves a centralized entity. It does not make sense in a local app such as rotki.

**Reply by kelsos** [reply]
For the electron app certainly, it doesn't make sense.

**Comment by LefterisJP** [comment]
Idea does not make sense for the electron app
